### PR TITLE
removed api/v1beta and updated dependency versions

### PR DESCRIPTION
**Pull request by bisztray**

### CODE REVIEW

File: application-lib/pkg/kubelib/deployment_helpers.go [hunk 1]
```diff
@@ -4,12 +4,12 @@
 
 package kubelib
 
-import apps "k8s.io/api/apps/v1beta1"
+import apps "k8s.io/api/apps/v1"
 
 func CreateDeployment(name string) *apps.Deployment {
```
Comment (howardhalili) [inline application-lib/pkg/kubelib/deployment_helpers.go:9]: this seems like duplicate of func CreateDeploymentConfig application-lib/pkg/kubelib/api.go

File: application-lib/pkg/kubelib/deployment_helpers.go [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
 	dep := &apps.Deployment{}
 	dep.Kind = "Deployment"
-	dep.APIVersion = "extensions/v1beta1"
+	dep.APIVersion = "extensions/v1"
```
Comment (howardhalili) [inline application-lib/pkg/kubelib/deployment_helpers.go:12]: Deployment is in apps/v1 API version https://kubernetes.io/docs/reference/using-api/deprecation-guide/#deployment-v116, https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#deployment-v1-apps

File: application-lib/pkg/kubelib/deployment_helpers.go (hunk 1 above)
Comment (bisztray) [reply application-lib/pkg/kubelib/deployment_helpers.go]: it apparently does, so removed this one

File: application-lib/pkg/kubelib/deployment_helpers.go (hunk 2 above)
Comment (bisztray) [reply application-lib/pkg/kubelib/deployment_helpers.go]: removed it, so this should be resolved

**Review by howardhalili — APPROVED** [review]

**Review by hochdorf — APPROVED** [review]

**Pull request merged**